remix/app/repositories/user.server.test.ts

```diff
@@ -0,0 +1,55 @@
+import type { User } from "@prisma/client";
+import { prismaMock } from "../tests/setup-test-env";
+import UserRespository from "../repositories/user.server"
+
+describe("Test user model", () => {
```

Comment:
This should describing the object under test:

```suggestion
describe("User Repository", () => {
```

Reply:
Fixed.